frontend/src/components/worksheets/Worksheet/Worksheet.js

```diff
@@ -778,28 +863,44 @@ class Worksheet extends React.Component {
                 }.bind(this),
             );
         }
+        // paste after current focused cell
+        if (this.state.ws.info.edit_permission) {
+            Mousetrap.bind(
+                ['a v'],
+                function(e) {
+                    this.pasteToWorksheet();
+                }.bind(this),
+                'keyup',
+            );
+        }
 
         if (this.state.showBundleOperationButtons) {
             // Below are allowed shortcut even when a dialog is opened===================
             // The following three are bulk bundle operation shortcuts
             Mousetrap.bind(['backspace', 'del'], () => {
-                if (this.state.openDetach || this.state.openKill) {
+                if (this.state.openDetach || this.state.openKill || this.state.openCopy) {
                     return;
                 }
                 this.togglePopupNoEvent('rm');
             });
             Mousetrap.bind(['a d'], () => {
-                if (this.state.openDelete || this.state.openKill) {
+                if (this.state.openDelete || this.state.openKill || this.state.openCopy) {
                     return;
                 }
                 this.togglePopupNoEvent('detach');
             });
             Mousetrap.bind(['a k'], () => {
-                if (this.state.openDetach || this.state.openDelete) {
+                if (this.state.openDetach || this.state.openDelete || this.state.openCopy) {
                     return;
                 }
                 this.togglePopupNoEvent('kill');
             });
+            Mousetrap.bind(['a c'], () => {
```

Comment:
Can you update the PR description and the "keyboard shortcuts" popup/documentation since you've now implemented `a c` and `a v`?